datalake-api/src/main/java/com/sequenceiq/sdx/api/endpoint/SdxEndpoint.java

```diff
@@ -206,6 +206,20 @@ SdxBackupStatusResponse backupDatalakeStatusByName(@PathParam("name") String nam
             @QueryParam("backupId") String backupId,
             @QueryParam("backupName") String backupName);
 
+    @GET
+    @Path("/datalakename/{datalakeName}/backupDatalakeStatus")
```

Comment:
I think we should keep the url convention for the REST endpoints.
I see the original endpoint `{name}/backupDatalakeStatus` uses the backupId, so I suggest something like this for the new endpoints:
`{name}/backupStatus` or `{name}/backupStatusByDatalakeName`
and
`{name}/backupStatusByBackupId/{backupId}`

Reply:
These were introduced originally next to [@Path("{name}/backupDatalakeStatus")](https://github.com/hortonworks/cloudbreak/blob/master/datalake-api/src/main/java/com/sequenceiq/sdx/api/endpoint/SdxEndpoint.java#L201-L207). If you check, for example the [SdxUpgradeEndpoint](https://github.com/hortonworks/cloudbreak/blob/master/datalake-api/src/main/java/com/sequenceiq/sdx/api/endpoint/SdxUpgradeEndpoint.java) there are also similar operation id implementations:
```
 @Path("{name}/upgrade")
...
@Path("/crn/{crn}/upgrade")
```

Reply:
I see that there is a convention for crn's, but in _SdxEndpoint_ all other methods that use the datalake name, use the same **{name}/operation** format.
Also the datalake name is required in all requests, it can be confusing to have @PathParam in one request and @QueryParam in another.